### PR TITLE
T/matt 2060 add pause parent request

### PR DESCRIPTION
**Pull request by spiraleddy**
ref MATT-2060 Gary King's crew requests the pause notification from parent to player in iframe.

### CODE REVIEW

File: app-src/set-up-parent-frame-communications.js [hunk 1]
```diff
@@ -21,7 +22,11 @@ function setUpParentFrameCommunications(document) {
 }
 
 function playVideos() {
-  paella.player.videoContainer.play();
+  $(document).trigger(paella.events.play);
```
Comment (lbjay) [inline app-src/set-up-parent-frame-communications.js]: Can you explain the shift to the indirect triggering here?

Comment (spiraleddy) [inline app-src/set-up-parent-frame-communications.js]: The triggering of the play event notifies the VideoContainer to start playing, and also notifies plugs such as the PlayPauseButtonPlugin. It is the normal way a plugin communicates the play request. 

I'm not sure why Jim chose to call the play function directly, but calling it does not trigger a play event and prevents other UI components from knowing that the video is playing.

Comment (djcp) [inline app-src/set-up-parent-frame-communications.js]: Oh. I think he (or maybe you) mentioned this specifically as a limitation in his implementation, so I bet @karendolan's solution here is the correct one.

**Comment by lbjay** [comment]
Cool beans. :+1:

**Comment by spiraleddy** [comment]
Forgot to merge this!

**Pull request merged**